Formula/liquigraph.rb

```diff
@@ -0,0 +1,22 @@
+class Liquigraph < Formula
+  desc "Migration runner for Neo4j"
+  homepage "http://www.liquigraph.org"
+  url "https://github.com/fbiville/liquigraph/archive/liquigraph-1.0-RC3.tar.gz"
```

Comment:
Tagged as an upstream issue for now so you don't get nagged too frequently on the status here.